### PR TITLE
2016-06-30 and 2016-07-14 meeting minutes

### PR DESCRIPTION
**Pull request by williamkapke**
Please review. It would be nice to get a cadence of accepting these regularly.

### CODE REVIEW

File: meetings/2016-07-14.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+* Regular roll-up reporting for active working groups [#109](https://github.com/nodejs/TSC/issues/109)
+* Add @nebrius as a member of the TSC [#108](https://github.com/nodejs/TSC/pull/108)
+* Define "Node.js core" [#84](https://github.com/nodejs/TSC/issues/84)
+
+
+## Standup
+
+
+* Michael Dawson: Made the new “Define Node.js Core” PR
```
Comment (mhdawson) [inline meetings/2016-07-14.md]: I'd update this to 
* Michael Dawson: Some interaction and pushed minor changes to "Define Node.js Core" PR

**Comment by mhdawson** [comment]
LGTM with one minor update as suggested above.

**Comment by williamkapke** [comment]
Thanks for reviewing @mhdawson. I made the change you wanted.

**Comment by williamkapke** [comment]
@jasnell how 'bout a LGTM here so I can merge them ;)

**Comment by jasnell** [comment]
LGTM! :-)

**Pull request merged**